aya/src/bpf.rs

```diff
@@ -234,18 +254,16 @@ impl<'a> BpfLoader<'a> {
     /// # Example
```

Comment:
Personally, I think it would be helpful to show the examples from your commit message for thing1 and vec![1u8] because I think these are likely to be real-world use cases, but I usually make my examples more complicated than people want, so I'll leave it up to the rest of you.